### ISSUE TITLE
import with extra arguments fails due to argument order

### ISSUE THREAD

**Issue by bdashrad** [gruntwork-io/terragrunt#142]
Using the following config, import does not work as expected, due to resulting argument order:

```
terragrunt= {
  terraform = {
    {
      extra_arguments "secrets" {
        arguments = [
          "-var-file=terraform.tfvars",
          "-var-file=terraform-secret.tfvars"
        ]
        commands = [
          "apply",
          "plan",
          "import",
          "push",
          "refresh"
        ]
      }
    }
  }
```

```
$ terragrunt import module.legacy.aws_elb.elb staging-balancer
[terragrunt] 2017/02/21 12:50:35 Reading Terragrunt config file at /Users/bradclark/codeterraform/environments/staging/terraform.tfvars
[terragrunt] 2017/02/21 12:50:35 Remote state is already configured for backend s3
[terragrunt] 2017/02/21 12:50:35 Attempting to acquire lock for state file staging in DynamoDB
[terragrunt] 2017/02/21 12:50:35 Attempting to create lock item for state file staging in DynamoDB table terragrunt_locks
[terragrunt] 2017/02/21 12:50:36 Lock acquired!
[terragrunt] 2017/02/21 12:50:36 Running command: terraform import module.legacy.aws_elb.elb staging-balancer -var-file=terraform.tfvars -var-file=staging.tfvars
The import command expects two arguments.
Usage: terraform import [options] ADDR ID
```

The extra arguments (options) need to be passed between `terraform import` and the resources to import, like this:

```
$ terraform import -var-file=terraform.tfvars -var-file=staging.tfvars module.legacy.aws_elb.elb staging-balancer
module.legacy.aws_elb.elb: Importing from ID "staging-balancer"...
module.legacy.aws_elb.elb: Import complete!
  Imported aws_elb (ID: staging-balancer)
module.legacy.aws_elb.elb: Refreshing state... (ID: staging-balancer)

Import success! The resources imported are shown above. These are
now in your Terraform state. Import does not currently generate
configuration, so you must do this next. If you do not create configuration
for the above resources, then the next `terraform plan` will mark
them for destruction.
```

**Comment by brikis98** [comment]
Ah, good catch. I suspect there are probably other commands that have a similar issue. PRs to fix this are very welcome!

**Comment by brikis98** [comment]
Should be fixed by #205.
